sensor_msgs/Image img1
sensor_msgs/Image img2
---
bool isSimilar
float64 similar
bool error